vernac/ppvernac.ml

```diff
@@ -1283,5 +1283,5 @@ let pr_vernac_attributes =
       return (keyword "Fail" ++ spc() ++ pr_vernac_control v)
 
     let pr_vernac v =
-      try pr_vernac_control v
+      try pr_vernac_control v.v
```

Comment:
Any reason why pr_vernac_control doesn't take the cast?

Reply:
Fixed thanks.